datascience/tables.py

```diff
@@ -1599,29 +1602,32 @@ def with_column(self, label, values, *rest):
         """
         # Ensure that if with_column is called instead of with_columns;
         # no error is raised.
-        if rest:
-            return self.with_columns(label, values, *rest)
 
         new_table = self.copy()
-        new_table.append_column(label, values)
+        if formatter == {}:
+            formatter = None
+        elif isinstance(formatter, dict):
+            formatter = formatter["formatter"]
+        new_table.append_column(label, values, formatter)
         return new_table
 
-    def with_columns(self, *labels_and_values):
+    def with_columns(self, *labels_and_values, **formatter):
         """Return a table with additional or replaced columns.
 
 
         Args:
-            ``labels_and_values``: An alternating list of labels and values or
-                a list of label-value pairs. If one of the labels is in
-                existing table, then every value in the corresponding column is
+            ``labels_and_values``: An alternating list of labels and values or a list of label-value pairs. 
+                If one of the labels is in existing table, then every value in the corresponding column is
                 set to that value. If label has only a single value (``int``),
                 every row of corresponding column takes on that value.
+            ''formatter'' (single Formatter value): A single formatter value that will be applied to all columns 
+                being added using this function call.
 
         Raises:
             ``ValueError``: If
                 - any label in ``labels_and_values`` is not a valid column
                     name, i.e if label is not of type (str).
-                - if any value in ``labels_and_values`` is a list/array and
+                - if any value in ``labels_and_values`` is a list/array and`
```

Comment:
Is that extra backtick extraneous?

Reply:
Wow, what a catch! Yes, it was - pushing the changes now!